src/zm_config.h.in

```diff
@@ -65,6 +65,8 @@ struct StaticConfig
     std::string DB_USER;
     std::string DB_PASS;
     std::string PATH_WEB;
+	std::string SERVER_NAME;
+	unsigned int SERVER_ID;
```

Comment:
Do we maybe want to make SERVER_ID an unsigned char?  256 should be plenty of server IDs, no?